PROTOCOL.md

```diff
@@ -95,6 +95,7 @@ Here is an example of a Delta table with three entries in the commit log, stored
 /mytable/_delta_log/_last_checkpoint
 /mytable/_change_data/cdc-00000-924d9ac7-21a9-4121-b067-a0a6517aa8ed.c000.snappy.parquet
 /mytable/part-00000-3935a07c-416b-4344-ad97-2a38342ee2fc.c000.snappy.parquet
+/mytable/deletion_vector-0c6cbaaf-5e04-4c9d-8959-1088814f58ef.bin
```

Comment:
are deletion vectors are always in the root directory? they are not in some subdirectory like `_change_data` ?

Reply:
That is the proposal, yes. They are essentially required to read the data files, so imo it makes sense to store them alongside the data files. Except not taking partition hierarchy into account due to the "many DVs per file"-thingy.